Examples/WebViewExample/Podfile

```diff
@@ -0,0 +1,8 @@
+use_frameworks!
+
+pod "Branch", path: "../.."
```

Comment:
For the other examples I removed the pod spec for Branch since I wanted the examples to use the same version of the SDK as the current example.  Instead I included a reference to the Branch SDK in the project.

I like this solution too.  Do you think having the ' path: "../.."' spec might confuse beginners?